moveit_setup_assistant/src/widgets/navigation_widget.cpp

```diff
@@ -124,64 +124,48 @@ QSize NavDelegate::sizeHint(const QStyleOptionViewItem& option, const QModelInde
 void NavDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const
 {
   const bool isSelected = option.state & QStyle::State_Selected;
+  const QPalette& palette = QApplication::palette();
 
-  // NavScreen tp = index.data().value<NavScreen>();
-  QString nav_name = index.data().value<QString>();
+  QString nav_name = displayText(index.data(), option.locale);
 
   painter->save();
 
-  QLinearGradient backgroundGradient(QPoint(option.rect.x(), option.rect.y()),
-                                     QPoint(option.rect.x(), option.rect.y() + option.rect.height()));
+  // draw background gradient
+  QLinearGradient backgroundGradient(option.rect.topLeft(), option.rect.bottomLeft());
   if (isSelected)
   {
-    backgroundGradient.setColorAt(0, QColor(109, 164, 219));
-    backgroundGradient.setColorAt(1, QColor(61, 138, 212));
+    backgroundGradient.setColorAt(0, palette.color(QPalette::Highlight).lighter(125));
+    backgroundGradient.setColorAt(1, palette.color(QPalette::Highlight));
     painter->fillRect(option.rect, QBrush(backgroundGradient));
   }
   else
   {
-    backgroundGradient.setColorAt(0, QColor(245, 245, 245));
-    backgroundGradient.setColorAt(1, QColor(240, 240, 240));
+    backgroundGradient.setColorAt(0, palette.color(QPalette::Light));
+    backgroundGradient.setColorAt(1, palette.color(QPalette::Light).darker(125));
```

Comment:
make this darker amount less?

Reply:
@davetcoleman I am to blame here, because I tuned those values. 
Of course, that's a matter of taste, but I liked the 3D-look of the gradient. In your original version, the gradient was only barely visible and I suggest to drop the gradient completely if you want to keep it as is (That's what @VictorLamoine did after his rework towards palette colors).
If you like to keep the gradient, however, please play around with the argument to `.darker(xxx)` yourself to find a value the matches your taste. I actually don't care...

Reply:
The point of this code change is to make MSA look nice on most platforms, not just on one particular theme or desktop environment so I agree with you Dave.

 I will try to tweak those values and test the results on multiple environments to get the best compromise :)